src/gsp.py

```diff
@@ -0,0 +1,15 @@
+"""Get GSP boundary data from eso """
+import geopandas as gpd
+from nowcasting_dataset.data_sources.gsp.eso import get_gsp_shape_from_eso
+
+
+def get_latitude_longitude_gsp_boundaries_from_eso() -> gpd.GeoDataFrame:
+    """Get GSP boundaries"""
```

Comment:
nit. I think it would be good to add the lat/lon here again

```suggestion
    """Get GSP boundaries in lat/lon format (EPSG:4326)"""
```